.scripts/gen-option/gen-option.ts

```diff
@@ -1,16 +1,13 @@
-import {
-  difference,
-  intersection,
-} from "https://deno.land/x/set_operations@v1.1.1/mod.ts";
-import * as path from "https://deno.land/std@0.217.0/path/mod.ts";
+import { difference, intersection } from "@lambdalisue/set-operations";
```

Comment:
`Set#difference()` and `Set#intersection()` have been available since Deno v1.42.

https://deno.com/blog/v1.42#v8-123-and-typescript-543